src/_includes/head.html

```diff
@@ -93,6 +113,8 @@
 <link rel="stylesheet" href="{{ '/assets/custom/css/custom-navbar-or-navigation.css' | prepend: site.baseurl }}">
 <link rel="stylesheet" href="{{ '/assets/custom/css/custom-layout-colours.css' | prepend: site.baseurl }}">
 <link rel="stylesheet" href="{{ '/assets/custom/css/custom-layout-buttons.css' | prepend: site.baseurl }}">
+<link rel="stylesheet" href="{{ '/assets/custom/css/custom-softmod-feature.css' | prepend: site.baseurl }}">
+<link rel="stylesheet" href="{{ '/assets/custom/css/custom-softmod-tabs.css' | prepend: site.baseurl }}">
```

Comment:
So many different CSS files. I see that this PR wasn't the start of it but we need to work on bundling these things.

Reply:
It's very convoluted. We are battling against two boilerplate unity themes. The whole thing needs to be inspected and potentially rebuilt from the ground up

Reply:
Yep. We definitely need to look at how to resolve these issues. I left my comments to try to get that conversation started. @keith-smale suggested that perhaps now isn't the time to take this on – it should be a project in it's own right. Hopefully next week I'll have some time to start making suggestions about how we do that. 

I'd prefer an iterative approach to a clean break if we can come up with a straight forward strategy for doing it.